share/transformers/org_datacite.py

```diff
@@ -551,12 +556,12 @@ def get_schema(self, type):
     tags = tools.Map(
         tools.Delegate(ThroughTags),
         tools.RunPython(
-            force_text,
+            'text_list',
             tools.Concat(
                 tools.Maybe(tools.Maybe(ctx.record, 'metadata')['oai_datacite'], 'type'),
                 tools.RunPython(
                     'text_list',
-                    (tools.Concat(tools.Try(ctx.record.metadata['oai_datacite'].payload.resource.subjects.subject)))
```

Comment:
If `Concat` was the problem for tags, it should be removed for the subjects as well (https://github.com/CenterForOpenScience/SHARE/pull/650/files#diff-bedec94698aaa5a42507c84db3ae44d1R549).